src/Hal/Application/Command/Job/DoAnalyze.php

```diff
@@ -101,18 +100,18 @@ public function execute(ResultCollection $collection, ResultCollection $aggregat
 
         // tools
         $classMap = new ClassMap();
-        $tokenizer = new Tokenizer(new CacheMemory());
```

Comment:
yes... in the current master branch. But once this pull request is merged the code does not need it. (I searched through the whole code base: "CacheMemory" is not mentioned anywhere)